tools/goctl/api/parser/g4/ast/apiparser.go

```diff
@@ -3,6 +3,7 @@ package ast
 import (
 	"fmt"
 	"io/ioutil"
+	path2 "path"
```

Comment:
only one path in imports, change the `path` variable to other names. and keep import as path.